### DISCUSSION TITLE
Multiple TriPower with one hub possible?

### DISCUSSION THREAD

**Discussion by hoffigk** [AnotherDaniel/smahub#48] [General]
Hello,

First of all, thank you for the great software you have created.

I have two SMA Tripower and one SMA HomeManager. The SmaHub is currently supplying data from the manager. I would now like to integrate the two Tripower, but there is only one environment variable block TRIPOWERX_* with which I can only integrate one Tripower.

Can you tell me whether it is currently possible to integrate two Tripower via a SmaHub? I may have overlooked something.

Starting a second hub would of course also be possible, but I'd better ask first.

Thank you

Translated with DeepL.com (free version)

**Comment by AnotherDaniel** [comment]
Hi, and thank you for taking the time to post the question!

I confess that when I wrote this, the option of more than one inverters existing in a system didn't even occur to me (I have a single-house installation myself). 
So currently smahub does not support this scenario. Your request will go onto the feature list, but at the moment I cannot make promises on when I might be able to implement this...

**Comment by hoffigk** [comment]
Hi,

Thank you for your quick reply. I am very happy to hear from you. As I mentioned, I just wanted to know if it is possible to start multiple instances. Since I use Docker Swarm, I can start multiple SMA Hub instances easily.

By the way, I found out that the Tripower plugin does not work with the "Sunny Tripower 5.0". The calls to the API always get a 400 request error. I might need to use the same API that the frontend uses.
